src/DDTrace/OpenTelemetry/Span.php

```diff
@@ -475,4 +517,32 @@ private function updateSpanLinks()
         $this->links = $otel;
         $this->totalRecordedLinks = count($otel);
     }
+
+    private function updateSpanEvents()
+    {
+        $datadogSpanEvents = $this->span->events;
+        $this->span->meta["events"] = count($this->events);
+
+        $otel = [];
+        foreach ($datadogSpanEvents as $datadogSpanEvent) {
+            // Check if the event relationship exists
+            $event = ObjectKVStore::get($datadogSpanEvent, "event");
+            if ($event === null) {
+                // Create the event
+                $event = new Event(
+                    $datadogSpanEvent->name,
+                    (int)$datadogSpanEvent->timestamp,
+                    Attributes::create((array)$datadogSpanEvent->attributes ?? [])
```

Comment:
This creates an OTel-Event, but it won't include the exception. Can you generate the right exception-attributes here on the fly? if $datadogSpanEvent instanceof ExceptionSpanEvent? (i.e. exception.stacktrace, exception.type, exception.message)?

Reply:
I.e. what you do in SpanBuilder::recordException